public/mockServiceWorker.js

```diff
@@ -62,7 +62,12 @@ self.addEventListener('message', async function (event) {
 
       sendToClient(client, {
         type: 'MOCKING_ENABLED',
-        payload: true,
+        payload: {
```

Comment:
my IDE is suggesting a warning: "`Missing await for an async function call `" what you think?